src/app/components/FrostedGlassPromo/FrostedGlassPanel.tsx

```diff
@@ -0,0 +1,59 @@
+/** @jsxRuntime classic */
```

Comment:
I think it's worth adding this to the component standards after, even just if it's temporary to prevent builds failing

Reply:
Yea will need documented somewhere as this will be needed in every new TSX file going forward that makes use of the Emotion CSS prop.